mgmtfn/k8splugin/types.go

```diff
@@ -0,0 +1,451 @@
+package k8splugin
+
+import "time"
+
+// ListMeta describes metadata that synthetic resources must have, including lists and
+// various status objects.
+type ListMeta struct {
```

Comment:
We can't include the package as is. There are just too many dependencies. Cloning just what we need seems to be the standard practice around this, judging from other similar packages.